charts/fluent-operator/values.yaml

```diff
@@ -89,7 +69,20 @@ output:
       enable: false
       brokers: "<kafka broker list like xxx.xxx.xxx.xxx:9092,yyy.yyy.yyy.yyy:9092>"
       topics: ks-log
-  fluentd:
+
+fluentd:
+  enable: false
```

Comment:
we need resources for fluentd as well, @zhu733756 do you have any suggest values?

Reply:
> we need resources for fluentd as well, @zhu733756 do you have any suggest values?

I have some suggests below, totally lgtm.